pkg/api/store/cluster/cluster_store.go

```diff
@@ -313,3 +320,19 @@ func validateNetworkFlag(data map[string]interface{}, create bool) error {
 
 	return nil
 }
+
+func enableLocalBackup(data map[string]interface{}) {
+	rkeConfig, ok := values.GetValue(data, "rancherKubernetesEngineConfig")
+
+	if ok && rkeConfig != nil {
+		backupConfig := values.GetValueN(data, "rancherKubernetesEngineConfig", "services", "etcd", "backupConfig")
+		if backupConfig == nil {
+			backupConfig = &v3.BackupConfig{
```

Comment:
just wanted to confirm if the backupConfig value doesn't need an explicit conversion into map, and gets saved as intended on the rke config

Reply:
it's stored correctly in the config.